src/test/java/org/broadinstitute/hellbender/engine/dataflow/RefAPISourceUnitTest.java

```diff
@@ -112,6 +113,28 @@ public void testReferenceSourceQueryWithNullInterval() {
     }
 
 
+    @Test(groups = "cloud")
+    public void testReferenceSourceMultiPageQuery() {
+        final int mio = 1_000_000;
```

Comment:
Since this test only makes sense if it matches the page size,  make `ReferenceAPISource.pageSize` visible and use it directly.